spec/core/plugin_spec.rb

```diff
@@ -29,6 +29,22 @@ class LogStash::Filters::LadyGaga < LogStash::Filters::Base
     expect(LogStash::Plugin.lookup("filter", "lady_gaga")).to eq(LogStash::Filters::LadyGaga)
   end
 
+  describe "#inspect" do
+    class LogStash::Filters::MyTestFilter < LogStash::Filters::Base
```

Comment:
I can't seem to properly use a let here to create a class. I can use a 

```
let(:new_filter_class) { Class.new(Filters::Base)  do ... end }
````

but then the new_filter_class will not have a name I can later match against in the spec because it's an anonymous class.
I'd then consider leaving this as is.